03-transactions.md

```diff
@@ -13,7 +13,7 @@ Most transaction outputs used here are P2WSH outputs, the segwit version of P2SH
 ## Funding Transaction Output
 
 * The funding output script is a pay-to-witness-script-hash<sup>[BIP141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#witness-program)</sup> to:
-   * `0 2 <key1> <key2> 2 OP_CHECKMULTISIG`
+   * `2 <key1> <key2> 2 OP_CHECKMULTISIG`
```

Comment:
Yep, good catch!  I expect more of these as we implement and create test vectors...